students/README.md

```diff
@@ -0,0 +1,86 @@
+# CNCF Students toolkit
+Is a group of resources, meeting ideas, and groups to get students involved in the CNCF ecosystem. They will be able to gain experience using cloud native technologies for their future jobs and develop skills to contribute in open source projects. It also creates a space for students and universities in the CNCF ecosystem.
+ ## How it works
+There are many different options that a student can choose to work with or learn about CNCF projects, the following sections describe a few of the different options.
+ 
+### Working on issues
+College students (primarily engineering students) have the task of working on issues of their interest.
+ 
+### Working on thesis
+A college student can work on a research topic for their own thesis that involves the implementation, or a test drive of cloud native technologies that solve real life problems, or shows different use cases for cloud native technologies.
+ 
+### Class projects
+A Professor can design a project that solves a real work problem and show how to uses cloud native technologies, the professor can explore ideas such as:
+- Test new features for an specific CNCF project
+- Implement a solution that solves a real life problem
+- Develop new features for a CNCF project
+ At the end, the best projects of the class can be presented to CNCF projects or into CNCF Community Groups. In addition, these projects develop skills that student need to apply to the Google Summer of Code.
+ 
+### Speakers and other contributions
+These initiatives should not be only focused on coding. Students can also be speakers or social media contributors, like the [CNCF Ambassador Program](https://github.com/cncf/ambassadors).
+ 
+### Open Syllabus
+Professors can work on open content of their courses and start to include CNCF technologies in their classes and projects, generating content as workshops, tutorials, etc., created by professors or students.
+ 
+### CNCF Students Day
+Professors and Universities can organize events to introduce students into CNCF technologies, including basic content as how to use containers, Kubernetes and complementary cloud native technologies. They can also attend [Cloud Native Community Groups](https://community.cncf.io/) or [Kubernetes Community Days](https://kubernetescommunitydays.org/) in their region. Many of these events have free or discounted tickets.
```

Comment:
Some suggestion to make sentence easier to read.

- *to CNCF technologies*
- *content such as containers*